tensorflow/core/kernels/constant_op.cc

```diff
@@ -50,6 +50,7 @@ void ConstantOp::Compute(OpKernelContext* ctx) { ctx->set_output(0, tensor_); }
 ConstantOp::~ConstantOp() {}
 
 REGISTER_KERNEL_BUILDER(Name("Const").Device(DEVICE_CPU), ConstantOp);
+REGISTER_KERNEL_BUILDER(Name("Const").Device(DEVICE_SYCL), ConstantOp);
```

Comment:
Should these be protected by "TENSORFLOW_USE_SYCL" ?  Otherwise we're registering constants for devices that don't exist.